### ISSUE TITLE
What works?

### ISSUE THREAD

**Issue by Erudition** [miniBill/elm-prelude#1]
Interested in knowing what parts are currently working.

**Comment by miniBill** [comment]
Http requests, basic layouts (row/column), mouse support and what I needed from the core lib.
I got stuck on writing a layout algorithm for tables and then got carried away by other projects.
Are you interested in contributing / using this?

Have a look at the `elm-tea-example` directory for... examples =D

**Comment by Erudition** [comment]
Absolutely! Oh and I researched the Extensible Records problem today. Turns out people have already added them to Haskell!

https://wiki.haskell.org/Extensible_record

for example, [this library, CTRex](https://wiki.haskell.org/CTRex) implements it. 
I was thinking we could wrap that somehow?
